### PR TITLE
Add $pageTestPrep

### PR DESCRIPTION
**Pull request by helenemccarron**
Add the end of chapter mixin for Test Prep For AP® Courses  and documentations

### CODE REVIEW

File: books/rulesets/output/economics.css [hunk 1]
```diff
@@ -740,6 +763,15 @@
   container: h1;
   data-type: "document-title";
   content: pending(-TITLECONTAINER); }
+:pass(5) [data-type="composite-page"].os-eoc.os-ap-test-prep-container::before {
+  container: span;
+  content: "Test Prep For AP® Courses";
+  class: os-text;
+  move-to: -TITLECONTAINER; }
```
Comment (reedstrm) [inline books/rulesets/output/economics.css]: bucket name starts with `-` ?

Comment (reedstrm) [inline books/rulesets/output/economics.css]: did some map-key get typoed?

Comment (helenemccarron) [inline books/rulesets/output/economics.css]: This appears to be a bug in how the $bucket is used. See PR #72

File: books/rulesets/output/economics.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@charset "UTF-8";
```
Comment (philschatz) [inline books/rulesets/output/economics.css]: ooh, what caused this to show up in this file and not the other CSS files?

Comment (helenemccarron) [inline books/rulesets/output/economics.css]: I believe the use of the (r) character

Comment (philschatz) [inline books/rulesets/output/economics.css]: ah yep, Thanks!

**Comment by philschatz** [comment]
@helenemccarron do you mind if I add `TEAmacroeconap` to `./scripts/test`? That way those files get regenerated?

**Comment by helenemccarron** [comment]
Go for it!

**Pull request merged**